crates/viewer/re_ui/src/ui_layout.rs

```diff
@@ -95,15 +99,21 @@ impl UiLayout {
         self.data_label_impl(ui, string.as_ref())
     }
 
+    fn decorate_url(ui: &mut egui::Ui, text: &str, galley: Arc<egui::Galley>) -> egui::Response {
+        if url::Url::parse(text).is_ok() {
+            ui.re_hyperlink(text, text)
+        } else {
+            ui.label(galley)
+        }
+    }
```

Comment:
This should be more correct, as it will benefit from the fine-tuning we do in `data_label_impl`.

```suggestion
    fn decorate_url(ui: &mut egui::Ui, text: &str, galley: Arc<egui::Galley>) -> egui::Response {
        if url::Url::parse(text).is_ok() {
            ui.re_hyperlink(galley, text)
        } else {
            ui.label(galley)
        }
    }
```